test/powershell/Modules/Microsoft.PowerShell.Utility/Out-File.Tests.ps1

```diff
@@ -127,4 +127,9 @@ Describe "Out-File" -Tags "CI" {
 	# reset to not read only so it can be deleted
```

Comment:
You didn't cause it, but since you are modifying the file, I have a small refactoring request for you. Please place lines 21-23 in a BeforeAll block like this:
``` powershell
    BeforeAll {
        $expectedContent = "some test text"
        $inObject = New-Object psobject -Property @{text=$expectedContent}
        $testfile = Join-Path -Path $TestDrive -ChildPath outfileTest.txt
    }
````